llvm/include/llvm/CodeGen/GlobalISel/GenericMachineInstrs.h

```diff
@@ -906,6 +906,16 @@ class GVScale : public GenericMachineInstr {
   };
 };
 
+/// Represents a step vector.
+class GStepVector : public GenericMachineInstr {
+public:
+  APInt getStep() const { return getOperand(1).getCImm()->getValue(); }
+
```

Comment:
Maybe also add a variant to return a uint64_t for convenience?

Reply:
I replaced the APInt by an uint64_t. The step is unsigned.